### DISCUSSION TITLE
Error while importing into ES6 module

### DISCUSSION THREAD

**Discussion by pawel-marciniak** [donmccurdy/glTF-Transform#375] [General]
Hi, I have problem while importing: 

`import { Document, NodeIO } from '@gltf-transform/core';`

it gives me:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/pawel/Sites/threejs-backend/node_modules/gl-matrix/dist/esm/vec3.js from /Users/pawel/Sites/threejs-backend/node_modules/@gltf-transform/core/dist/core.js not supported.
```

I have "type": "module" in my package.json, I tried everything and I don't know why I have this error. I'm using Node 16.10.0.

**Comment by donmccurdy** [comment]
Thanks for letting me know! Let's continue the discussion in https://github.com/donmccurdy/glTF-Transform/issues/376, it looks like an issue in a downstream dependency.
